src/libs/actions/Welcome/OnboardingFlow.ts

```diff
@@ -31,6 +32,17 @@ const onboardingLastVisitedPathConnection = Onyx.connect({
     },
 });
 
+let onboardingValues: Onboarding;
+Onyx.connect({
+    key: ONYXKEYS.NVP_ONBOARDING,
+    callback: (value) => {
+        if (value === undefined) {
+            return;
+        }
```

Comment:
why do we need this here?

Reply:
I saw the `Prefer an early return to a conditionally-wrapped function body` error in the tests there. Open to suggestions for nicer looking syntax 😅

Reply:
I think this is common in these callbacks so we could leave it like this

Reply:
I only see it in callbacks where the value is a boolean